src/main/java/TestOut.java

```diff
@@ -0,0 +1,43 @@
+import java.io.BufferedWriter;
+import java.nio.file.Path;
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.io.IOException;
+import java.nio.file.Paths;
+import java.nio.file.Files;
+import java.io.FileWriter;
```

Comment:
Your imports would adhere more to the coding standards if they were more consistent!